pyproject.toml

```diff
@@ -55,6 +55,7 @@ dependencies = [
     "flask==3.*",
     "tomlkit==0.13.*",
     "requests>=2.32.3",
+    "toml>=0.10.2",
```

Comment:
```suggestion
```

We're already using tomlkit elsewhere, so let's use tomlkit for this new functionality as well instead of adding another toml library.